server/api/url_check_handler.py

```diff
@@ -0,0 +1,14 @@
+from flask import jsonify, Blueprint, request
+from model import EventTypes
+
+url_check_handler = Blueprint('url_check_handler', __name__)
+
+
+@url_check_handler.route('/user/<uid>/is_unique', methods=['GET'])
+def url_is_unique(uid):
+    url = request.args.get('url')
+    if url is None:
+        return jsonify({'success': 'false', 'msg': 'Empty URL'}), 200
+    if EventTypes.query.filter_by(url=url).first():
```

Comment:
this is actually not the EventType URL.
So when I have an account, it would be:
`calenderapp.com/sina/event-type`
This URL checker is for the `sina` section which will be selected during onboarding